axelor-bank-payment/src/main/java/com/axelor/apps/bankpayment/service/bankreconciliation/BankReconciliationDomainServiceImpl.java

```diff
@@ -168,15 +169,18 @@ public String getCashAccountDomain(BankReconciliation bankReconciliation) {
   public String createDomainForMoveLine(BankReconciliation bankReconciliation)
       throws AxelorException {
     String domain = "";
-    List<MoveLine> authorizedMoveLines =
+    String idList =
         moveLineRepository
             .all()
             .filter(bankReconciliationQueryService.getRequestMoveLines())
             .bind(bankReconciliationQueryService.getBindRequestMoveLine(bankReconciliation))
-            .fetch();
+            .fetch(0, 0)
```

Comment:
Hello,
Problem here is that the move lines are still being fetched entirely (then mappped to get the Id), we want to avoid that by directly fetching the id  using the Query.select method